memory/migrate_pages.py

```diff
@@ -51,6 +51,8 @@ def setUp(self):
                         'Node list with memory: %s' % nodes)
 
         dist = distro.detect()
+        if (dist.name == 'rhel' and dist.version == '9'):
+            self.cancel("libhugetlbfs is not available RHEL 9.x onwards")
```

Comment:
@sasingh87 you mean the complete test is not supported from rhel9.x on-words?
because your cancelling the test for all rhel9.x

Reply:
Yes, we found out that  migrate_pages.py would NOT pass on versions  RHEL 9.0+.  Similar scenarios have been faced in the below Test :                                                 
   Reference: https://github.com/avocado-framework-tests/avocado-misc-tests/blob/master/memory/libhugetlbfs.py

Reply:
please add job.log or debug.log for more info.
would be great if you can add with both PASS and FAILED runs.

Reply:
The failed case generally gives following log:
FAILED Case:
--------------------
[root@xxxxx memory]# avocado run --test-runner runner migrate_pages.py
JOB ID     : c8051709613aa28ae40b0c9fc43e3827375fcd1d
JOB LOG    : /home/xxxxx/tests/results/job-2022-10-11T16.12-c805170/job.log
(1/1) migrate_pages.py:MigratePages.test: CANCEL: libhugetlbfs-devel is needed for the test to be run (2.63 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 1
JOB HTML   : /home/xxxxx/tests/results/job-2022-10-11T16.12-c805170/results.html
JOB TIME   : 3.15 s
[job_fail.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/9754340/job_fail.log)
[debug_fail.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/9754344/debug_fail.log)
----------------------------------------------
Once  fixed, the logs show up as :
PASS / Fixed Case:
---------------------
[root@xxxxx memory]# avocado run --test-runner runner migrate_pages.py
JOB ID     : b3df52601a08a2f99950cff78b3b065125147c4d
JOB LOG    : /home/xxxxx/tests/results/job-2022-10-11T16.16-b3df526/job.log
(1/1) migrate_pages.py:MigratePages.test: CANCEL: libhugetlbfs is not available RHEL 9.x onwards (0.46 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 1
JOB HTML   : /home/xxxxx/tests/results/job-2022-10-11T16.16-b3df526/results.html
JOB TIME   : 0.97 s